maxim/logger/mistral/client.py

```
@@ -0,0 +1,272 @@
+                    "name": generation_name,
+                    "model_parameters": MistralUtils.get_model_params(**logging_kwargs),
+                    "messages": MistralUtils.parse_message_param(messages),
+                }
+            )
+        except Exception as e:
+            scribe().warning(
+                f"[MaximSDK][MaximMistralChat] Error in generating content: {e}"
+            )
+
+        stream = await self._chat.stream_async(*args, **kwargs)
+        chunks: List[dict] = []
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)